specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.3/FormRecognizer.json

```diff
@@ -0,0 +1,2270 @@
+        },
+        "valueTime": {
+          "description": "Time value.",
+          "format": "time",
+          "type": "string",
+          "x-nullable": false
+        },
+        "valuePhoneNumber": {
+          "description": "Phone number value.",
+          "type": "string"
+        },
+        "valueNumber": {
+          "description": "Floating point value.",
+          "type": "number",
+          "x-nullable": false
+        },
+        "valueInteger": {
+          "description": "Integer value.",
+          "type": "integer",
+          "x-nullable": false
+        },
+        "valueArray": {
+          "description": "Array of field values.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/FieldValue"
+          }
+        },
+        "valueObject": {
+          "description": "Dictionary of named field values.",
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/FieldValue"
+          }
+        },
+        "valueSelectionMark": {
+          "description": "Selection mark value.",
+          "type": "string",
+          "enum": [
+            "selected",
+            "unselected"
+          ],
+          "x-nullable": false
+        },
+        "text": {
+          "description": "Text content of the extracted field.",
+          "type": "string"
+        },
+        "boundingBox": {
+          "description": "Bounding box of the field value, if appropriate.",
+          "$ref": "#/definitions/BoundingBox"
+        },
+        "confidence": {
+          "description": "Confidence score.",
+          "$ref": "#/definitions/Confidence"
+        },
+        "elements": {
+          "description": "When includeTextDetails is set to true, a list of references to the text elements constituting this field.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ElementReference"
+          }
+        },
+        "page": {
+          "description": "The 1-based page number in the input document.",
+          "type": "integer",
+          "minimum": 1,
+          "x-nullable": false
+        }
+      }
+    },
+    "FieldValueType": {
+      "type": "string",
+      "description": "Semantic data type of the field value.",
+      "enum": [
+        "string",
+        "date",
+        "time",
+        "phoneNumber",
+        "number",
+        "integer",
+        "array",
+        "object",
+        "selectionMark"
+      ],
+      "x-ms-enum": {
+        "name": "FieldValueType",
+        "modelAsString": false
+      },
+      "x-nullable": false
+    },
+    "Confidence": {
+      "description": "Confidence value.",
+      "type": "number",
+      "minimum": 0,
+      "maximum": 1,
+      "x-nullable": false
+    },
+    "Appearance": {
+      "description": "An object representing the appearance of the text line.",
+      "type": "object",
+      "required": [
+        "style",
+        "styleConfidence"
```

Comment:
Just curious if the changes to the shape of "Appearance" were intentional? This looked a bit different in preview.2 and defined a `Style` type that included the enum and confidence.

Reply:
yes.

Reply:
Could you expand more on the reasons why? This affects the design we have in the SDK which we were following according to the recommendations the REST API board gave the service.

Reply:
@bojunehsu had suggested this refactor to better capture the API intent. If this is impacting SDK side, please feel free to suggest alternative that works best.

Reply:
@sandeepparab Apologies for so going through so many design iterations.  The design in [preview.2](https://github.com/Azure/azure-rest-api-specs/blob/228cf296647f6e41182cee7d1a403990e6a8fe3c/specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.2/FormRecognizer.json#L2061) is correct and should remain unchanged in preview.3.  I recommend we copy the definitions for Appearance and Style from there.

Reply:
No worries @bojunehsu. Reverted to preview.2.